pkg/fatfs/Makefile.include

```diff
@@ -0,0 +1,2 @@
+INCLUDES += -I$(BINDIRBASE)/pkg/$(BOARD)
+DIRS += $(BINDIRBASE)/pkg/$(BOARD)/fatfs
```

Comment:
where else? The problem is the CFLAG must be added in any RIOT project that uses `fatfs`, adding these lines to the primary `Makefile` does not help (I tested btw.) because that one only downloads and patches the sources. But `Makefile.include` is recursivly added during build of any RIOT project, that's why it has to be in there.